README.md

```diff
@@ -10,21 +10,21 @@ These modules are distributed as [collections](https://docs.ansible.com/ansible/
 To install them, run:
 
 ```bash
-ansible-galaxy collection install t_systems_mms.letsencrypt
+ansible-galaxy collection install t_systems_mms.acme
 ```
 
 Alternatively put the collection into a `requirements.yml`-file:
 
 ```yaml
 ---
 collections:
-- t_systems_mms.letsencrypt
+- t_systems_mms.acme
 ```
 
 ## Usage
 
-Role `letsencrypt` for issuing certificates.
-Please see [documentation](docs/role-letsencrypt.md) for variables, usage and further information for all the different providers.
+Role `acme_letsencrypt` for issuing certificates.
```

Comment:
Shouldn't `letsencrypt` be removed here, too? The role can use other acme-providers, I guess.

Reply:
Hm... yeah you are right. I seem to have been a bit confused about the naming. Maybe we should name it acme-certificate in case we add other roles... or realy just acme?

Reply:
`acme` seems fine.

Reply:
Rename done